grpc-api/src/main/proto/gateway.proto

```diff
@@ -77,6 +77,16 @@ message FixtureResponse {
     repeated FixtureLine lines = 1;
 }
 
+/*
+ * Geese Types
+ */
+enum GooseType {
```

Comment:
Why do you need separate `GooseType` for gateway and geese service?

Reply:
Good point, will refactor it and put it into a shared proto.

Reply:
Pulled shared messages for geese and clouds into shared protos, created GeeseLocator  and CloudLocator messages for future extensibility